braininventory/get.py

```diff
@@ -99,4 +99,7 @@ def report():
 	report['is_reachable'] = df['URL'].apply(__is_reachable)
 
 	return report
+def __get_project_names(df):
```

Comment:
Good. You are only lacking documentation. Please document the function.

![](https://media2.giphy.com/media/j6wQILdEcwUqdy6MJH/giphy.gif?cid=ecf05e47koe99v14lacztkbmu6cumkexdkv9ajkgiqwdgcmr&ep=v1_gifs_search&rid=giphy.gif&ct=g)